src/components/FeedbackPage/FeedbackPage.scss

```diff
@@ -0,0 +1,31 @@
+@import '~style/variables';
+
+.feedbackWrapper {
+  @include container;
+
+  padding: 48px 0;
+  background: $color-overview--020;
+}
+
+.title {
+  margin-bottom: 40px;
+  font-size: 44px;
+  font-weight: 300;
+}
+
+.paragraph {
+  font-size: 22px;
+}
+
+.surveyFrame {
+  margin-top: 60px;
+  border: 1px solid $color-grey--500;
+  border-radius: 4px;
+  min-width: 100%;
+  min-height: 500px;
+
+  @include for-tablet-portrait-up {
+    min-width: 600px;
+    max-width: 800px;
```

Comment:
Is this as per the survey monkey specification? Could we simplify this, do away with the breakpoint and do:
```css
  width: 600px;
  max-width: 100%;
```